File: services/customers-service/src/main/java/br/com/deltaglobalbank/customers/features/customers/createCustomer/CreateCustomerResponse.java
package br.com.deltaglobalbank.customers.features.customers.createCustomer;

import java.time.Instant;
import java.time.LocalDate;
import java.util.UUID;

public record CreateCustomerResponse(
    UUID id,
    UUID tenantId,
    String cpf,
    String fullName,
    LocalDate birthDate,
    String status,
    Instant createdAt
) {
}

File: services/customers-service/src/main/kotlin/br/com/deltaglobalbank/customers/features/customers/createCustomer/CreateCustomerResponse.kt
package br.com.deltaglobalbank.customers.features.customers.createCustomer

import java.time.Instant
import java.time.LocalDate
import java.util.UUID

data class CreateCustomerResponse(
    val id: UUID,
    val tenantId: UUID,
    val cpf: String,
    val fullName: String,
    val birthDate: LocalDate,
    val status: String,
    val createdAt: Instant,
)